msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -380,6 +383,26 @@ protected PublicClientApplication(@NonNull final Context context,
         }
     }
 
+    private void setupTelemetry(@NonNull final Context context,
+                                @Nullable final PublicClientApplicationConfiguration developerConfig) {
+        if (null != developerConfig.getTelemetryConfiguration()) {
+            com.microsoft.identity.common.internal.logging.Logger.verbose(
+                    TAG,
+                    "Telemetry configuration is set. Telemetry is enabled."
+            );
+        } else {
+            com.microsoft.identity.common.internal.logging.Logger.verbose(
+                    TAG,
+                    "Telemetry configuration is null. Telemetry is disabled."
+            );
+        }
+
+        new com.microsoft.identity.common.internal.telemetry.Telemetry.Builder()
```

Comment:
shouldn't this be inside the if loop above where we check for `null != developerConfig.getTelemetryConfiguration()`

Reply:
developerConfig.getTelemetryConfiguration() is a nullable value here. If the developerConfig.getTelemetryConfiguration() is null means that the client does not set up the Telemetry configuration. In this case, the Telemetry will be disabled for this client.